### ISSUE TITLE
New Rule: Heading has non-empty accessible name

### ISSUE THREAD

**Issue by Jym77** [w3c/wcag-act#500]
The ACT-Rules community group has completed work on "Heading has non-empty accessible name". We would like to see this rule be added to the WCAG ruleset. You can find the rule here:

**Rule title**: Heading has non-empty accessible name
**Rule id**: ffd0e9
**Link to the rule**: https://act-rules.github.io/rules/ffd0e9
**License**: For community groups: W3C Software And Document Notice And License
**Implementors**:
- Alfa (Siteimprove)
- QualWeb (LASIGE, Faculdade de Ciências da Universidade de Lisboa)
- SortSite (PowerMapper)

If you'd like to contact me, please e-mail me at jym@siteimprove.com. (optional: You can also reach me on the W3C Community Slack, my name there is Jean-Yves Moyen)

**Comment by maryjom** [comment]
[Survey](https://www.w3.org/2002/09/wbs/93339/ACTHeadingName/) is open until 17 December.

**Comment by maryjom** [comment]
[Survey results](https://www.w3.org/2002/09/wbs/93339/ACTHeadingName/results) discussed at [7 January meeting](https://www.w3.org/2021/01/07-wcag-act-minutes.html#item02) and there were many concerns expressed. TF consensus was to not go forward with this rule as proposed.
